docs/source/cn/安装.md

```diff
@@ -0,0 +1,142 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
```

Comment:
Could you rename this file to its English name please? While the content must be in the target language (simplified Chinese) for the users, it is still better to have the file architecture in English so that the maintainers know which file contains what. Also, the `_toctree.yml` file is expecting English-based files (see `local: index` means "take the `index.md` local file).

This apply to other translated files and directories. Thanks in advance for making the changes!

Reply:
No problem, I'll modify it when I have some free time 🦾. Thanks! @Wauplin

Reply:
File name has been renamed to English @Wauplin